addons/point_of_sale/report/pos_order_report.py

```diff
@@ -74,6 +74,7 @@ def _from(self):
                 LEFT JOIN product_template pt ON (p.product_tmpl_id=pt.id)
                 LEFT JOIN uom_uom u ON (u.id=pt.uom_id)
                 LEFT JOIN pos_session ps ON (s.session_id=ps.id)
+                LEFT JOIN res_currency c ON (s.company_id=c.id)
```

Comment:
```suggestion
                LEFT JOIN res_company co ON (s.company_id=co.id)
                LEFT JOIN res_currency c ON (co.currency_id=c.id)
```

It seems that a `JOIN` is missing.
Otherwise, LGTM.